TypeCobol/Compiler/Diagnostics/Resources/DiagnosticMessages.csv

```diff
@@ -49,4 +49,6 @@ Category;Code;Severity;MessageTemplate;Document;PageNumber;ReferenceText
 2;49;1;The number of hexadecimal digits in binary-string constant must be even and must not exceed 32704.;0;0;Page
 2;50;1;The number of hexadecimal digits in graphic-string constant must be a multiple of 4 and must not exceed 32704.;0;0;Page
 2;51;1;The number of digits in the exponent of a decimal floating-point constant must not exceed 4.;0;0;Page
-2;52;1;The number of digits in the mantissa of a decimal floating-point constant must not exceed 34.;0;0;Page
\ No newline at end of file
+2;52;1;The number of digits in the mantissa of a decimal floating-point constant must not exceed 34.;0;0;Page
+2;53;2;Invalid character found before pseudo-text delimiter == : "{0}";1;47;An opening pseudo-text delimiter should be preceded by a space separator.
```

Comment:
Same remarks here. It'd be easier for our Cobol developers to read `==:` instead of `== :`.

Reply:
done